examples/gno.land/r/jjoptimist/config.gno

```diff
@@ -0,0 +1,51 @@
+package home
+
+import (
+    "std"
+)
+
+var (
+    
+    OWNER_ADDR = std.Address("g16vfw3r7zuz43fhky3xfsuc2hdv9tnhvlkyn0nj")
+
+    owner std.Address
+
+    config = struct {
+        title       string
+        description string
+        github     string
+    }{
+        title:       "JJOptimist's Home Realm 🏠",
+        description: "Exploring Gno and building on-chain",
+        github:      "jjoptimist",
+    }
+)
+
+func init() {
+    owner = OWNER_ADDR
```

Comment:
If you're defining this at top-level, you can simply initiate it where you declared it. 

This is a bit of a blurry line in Gno, so bear with me- following Go standards, if you can declare+initialize something immediately, do so at top level; `init` should be reserved for "doing" things you cannot do at declaration+initialization.